SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -1105,3 +1106,43 @@ void USpatialSender::UpdateInterestComponent(AActor* Actor)
 	Worker_EntityId EntityId = PackageMap->GetEntityIdFromObject(Actor);
 	Connection->SendComponentUpdate(EntityId, &Update);
 }
+
+void USpatialSender::RetireEntity(const Worker_EntityId EntityId)
+{
+	if (USpatialActorChannel* ActorChannel = NetDriver->GetActorChannelByEntityId(EntityId))
```

Comment:
Can you rework this to use the package map to get the Actor instead of using a channel - dormancy will change how this functions (can have an actor with an entity id, without a channel).

Reply:
will do